### ISSUE TITLE
Should be error message if any python modules are not found

### ISSUE THREAD

**Issue by ThemosTsikas** [ThemosTsikas/PhotoPolarAlign#2]
A window should pop up if any modules are not present, maybe telling the user how to install them.

**Comment by Steve102420** [comment]
Hi ThemosTsikas,

I would like use your script, because one of my favourite hobby is astro photographing.
But it isn't run for me. I installed all missing python module.
For example I got an error message: "PIL missing".
After I installed the library and the error will be fixed.

Unfortunatelly, now I don't get any error message. I don't know why.  The script was started, and I can chose the pictures but the image window not opening. I dont't get any info for the polar alignment offset error.
Can you help me?
I am trying this script, in WIN10, 2.7.1 Python.

In my opinion it is very usefoul program for astro photographers. I would like to try it.
Thanks in advance,
Steve

**Comment by ThemosTsikas** [comment]
Hello,

Thanks for getting in touch.
If you can choose the test images (v.jpg for the top one, h.jpg for the one
below that), does the Nova button become active? Do you have an API key
from nova.astrometry.net? If not, you should get one and put it in the
Settings.

Themos Tsikas

**Comment by Steve102420** [comment]
Yes I have API key. If I press Nova button I get an error message.
ImportErrorr: No module named ujson
Sorry, I haven't noticed this error so far.

**Comment by ThemosTsikas** [comment]
Hi,

Yes, I remember that. Use "conda install ujson" or whatever you used to
install python packages. You will also need pillow, scipy, astropy

Themos
